extensions/amp-story/1.0/amp-story.js

```diff
@@ -780,6 +780,10 @@ export class AmpStory extends AMP.BaseElement {
 
     this.storeService_.subscribe(StateProperty.SIDEBAR_STATE, sidebarState => {
       this.onSidebarStateUpdate_(sidebarState);
+      this.analyticsService_.triggerEvent(
```

Comment:
Please move this to within `onSidebarStateUpdate`. Keeping the listeners and handlers separated keeps the logic in one place, which helps with reusability.